pkg/reconciler/v1alpha1/route/table_test.go

```diff
@@ -78,7 +78,7 @@ func TestReconcile(t *testing.T) {
 				}},
 			}),
 		}},
-		WantErr: true,
+		WantErr: false,
```

Comment:
I'd just drop these, the default is false :)

Reply:
Done